src/common/utils/mediaHelpers.js

```diff
@@ -44,8 +44,23 @@ export const getMediaPath = ({ mediaObject, type, size = ORIGINAL }) => {
   }
 }
 
-export const getReadableFileSize = (size) =>
-  `${(size / (1024 * 1024)).toFixed(2)} MB`
+//reference https://www.codalas.com/en/2303/how-to-display-file-size-in-a-readable-format-in-javascript
+export const getReadableFileSize = (attachmentSize) => {
+  const DEFAULT_SIZE = 0
+  const fileSize = attachmentSize ?? DEFAULT_SIZE
```

Comment:
I don't think we need this line, since the first if condition will always handle the default case